feat: separate the sidebar from the player

pauldambra opened the pull request:
let's visually separate the sidebar from the player

so you can resize the playlist,player, and sidebar to get the layout you want

pauldambra committed:
feat: separate the sidebar from the player

pauldambra commented:
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.com/github/pr/PostHog/posthog/43346?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#43346** <a href="https://app.graphite.com/github/pr/PostHog/posthog/43346?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.com/github/pr/PostHog/posthog/43346?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#43340** <a href="https://app.graphite.com/github/pr/PostHog/posthog/43340?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#43339** <a href="https://app.graphite.com/github/pr/PostHog/posthog/43339?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `master`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

github-actions[bot] committed:
test(storybook): update UI snapshots

ksvat reviewed (approved):
[![hog_curator](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/7wPDwMTNWTS142S2Q9dG/f0f67556-d20f-4f94-8249-031b21549dbb.webp)](https://app.graphite.com/settings/meme-library?org=PostHog)

orian reviewed (approved):
LGTM for ClickHouse/SQL,

github-actions[bot] committed:
test(storybook): update UI snapshots

# Conflicts:
#	frontend/__snapshots__/replay-scenes-person--person-recording-tab-multiple-and-not-found--dark.png

github-actions[bot] committed:
test(storybook): update UI snapshots
